WordPressAuthenticator/Signin/LoginEmailViewController.swift

```diff
@@ -632,3 +633,9 @@ extension LoginEmailViewController: GoogleAuthenticatorLoginDelegate {
     }
 
 }
+
+extension LoginEmailViewController: ASWebAuthenticationPresentationContextProviding {
+    public func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
+        return view.window!
+    }
+}
```

Comment:
You'll see a number of these. They became necessary when I update `showFrom(viewController:` to required `viewController` to be `UIViewController & ASWebAuthenticationPresentationContextProviding`. 

It surprised me that so many `UIViewController` subclasses in were passed as parameters to that call and required the protocol conformance but haven't researched it further.